contracts/DAO.sol

```diff
@@ -8,15 +8,12 @@ import "./ModuleBase.sol";
 
 /// @notice A minimum viable DAO contract
 contract DAO is IDAO, ModuleBase {
-    string public name;
```

Comment:
Why was this `name` removed from the DAO contract?

Reply:
So the DAO contract will still have the name variable, since name was added to the ModuleBase, and DAO inherits from ModuleBase.